Userland/Libraries/LibWasm/AbstractMachine/Operators.h

```diff
@@ -289,8 +289,11 @@ struct VectorCmpOp {
         auto result = bit_cast<Native128ByteVectorOf<ElementType, SetSign>>(c1);
         auto other = bit_cast<Native128ByteVectorOf<ElementType, SetSign>>(c2);
         Op op;
-        for (size_t i = 0; i < VectorSize; ++i)
-            result[i] = op(result[i], other[i]) ? static_cast<MakeUnsigned<ElementType>>(-1) : 0;
+        for (size_t i = 0; i < VectorSize; ++i) {
+            SetSign<ElementType> lhs = result[i];
+            SetSign<ElementType> rhs = other[i];
```

Comment:
🤔 why would the signs be different? we already set them through the vector type

Reply:
To be honest with you... I have no idea why. I think it could be a number of things related to `NativeVectorType` and how the compiler does inference around it. Or we need to be explicit because the compiled native SIMD instructions aren't taking sign into account the way we wrote it before. 

This does make around 100 tests pass, though.

Reply:
Hmm
I'll take a look later today, and merge if there's no cleaner solution.